RELEASE.rst

```diff
@@ -63,6 +63,16 @@ and a `source distribution <https://packaging.python.org/guides/distributing-pac
 
      $ python3 -m twine upload dist/*
 
-#. Trigger a build of the docs on https://readthedocs.org/.
+#. Make sure the new version appears on https://motor.readthedocs.io/. If the
+    new version does not show up automatically, trigger a rebuild of "latest":
```

Comment:
The formatting in most of these lines looks off. For example this line needs to be indented only 3 spaces.

Reply:
Fixed